### PR TITLE
Added NL Translation

### PR DESCRIPTION
**Pull request by ThijmenGThN**
Country: Holland (The Netherlands)
 ~: Nederland, NL

Contact: Discord
 ~: ThijmenGThN#2684

### CODE REVIEW

File: kofta/public/locales/nl/translation.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+			"emotesSoon": "[emoticons binnenkort]",
+			"bannedAlert": "Je bent verbannen uit de chat",
+			"waitAlert": "U moet een seconde wachten voordat u weer een bericht verzendt",
+			"search": "Zoeken",
+			"searchResults": "Zoeken Resultaten",
+			"recent": "Vaak Gebruikt",
+			"sendMessage": "Een Bericht Versturen",
+			"whisper": "Vluisteren",
+			"welcomeMessage": "Welkom in de chat!",
+			"roomDescription": "kamer descriptie"
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
			"roomDescription": "kamer omschrijving"
```

File: kofta/public/locales/nl/translation.json [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+				"minLength": "min lengte 2"
+			}
+		},
+		"roomChat": {
+			"title": "Chat",
+			"emotesSoon": "[emoticons binnenkort]",
+			"bannedAlert": "Je bent verbannen uit de chat",
+			"waitAlert": "U moet een seconde wachten voordat u weer een bericht verzendt",
+			"search": "Zoeken",
+			"searchResults": "Zoeken Resultaten",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
			"searchResults": "Zoek resultaten",
```

File: kofta/public/locales/nl/translation.json [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+				"needsFuture": "moet in de toekomst zijn",
+				"roomName": "Kamer naam",
+				"minLength": "min lengte 2"
+			}
+		},
+		"roomChat": {
+			"title": "Chat",
+			"emotesSoon": "[emoticons binnenkort]",
+			"bannedAlert": "Je bent verbannen uit de chat",
+			"waitAlert": "U moet een seconde wachten voordat u weer een bericht verzendt",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
			"waitAlert": "Je moet een seconde wachten voordat u weer een bericht verzendt",
```
Tot nu toe wordt overal je gebruikt. Dan is het raar om hier ineens u te gebruiken.

File: kofta/public/locales/nl/translation.json [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+				"displayNameLabel": "Weergavenaam",
+				"usernameError": "lengte 4 tot 15 tekens en alleen alfanumeriek / onderstrepingsteken",
+				"usernameLabel": "Gebruikersnaam",
+				"bioError": "maximale lengte van 160 tekens",
+				"bioLabel": "Bio"
+			},
+			"profileModal": {
+				"blockUserConfirm": "Weet je zeker dat je wilt voorkomen dat deze gebruiker lid wordt van een kamer die je ooit hebt gemaakt?",
+				"blockUser": "block user",
+				"makeMod": "make moderator",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
				"makeMod": "moderator maken",
```

File: kofta/public/locales/nl/translation.json [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+				"unmod": "moderator opheffen",
+				"addAsSpeaker": "toevoegen als luidspreker",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json:180]:
```suggestion
				"addAsSpeaker": "toevoegen als spreker",
```

File: kofta/public/locales/nl/translation.json [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+				"moveToListener": "verplaatsen naar luisteraars",
+				"banFromChat": "verbannen van chatten",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
				"banFromChat": "verbannen van chat",
```

File: kofta/public/locales/nl/translation.json [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+				"displayNameError": "lengte 2 tot 50 karakters",
+				"displayNameLabel": "Weergavenaam",
+				"usernameError": "lengte 4 tot 15 tekens en alleen alfanumeriek / onderstrepingsteken",
+				"usernameLabel": "Gebruikersnaam",
+				"bioError": "maximale lengte van 160 tekens",
+				"bioLabel": "Bio"
+			},
+			"profileModal": {
+				"blockUserConfirm": "Weet je zeker dat je wilt voorkomen dat deze gebruiker lid wordt van een kamer die je ooit hebt gemaakt?",
+				"blockUser": "block user",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
				"blockUser": "block gebruiker",
```

File: kofta/public/locales/nl/translation.json [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+			},
+			"invitedToJoinRoomModal": {
+				"newRoomCreated": "Nieuwe Kamer Gemaakt",
+				"roomInviteFrom": "Kamer uitnodiging van",
+				"justStarted": "Zij zijn net begonnen",
+				"likeToJoin": ", wil je deelnemen?",
+				"inviteReceived": "je bent uitgenodigd voor"
+			},
+			"editProfileModal": {
+				"usernameTaken": "gebruikers naam niet beschikbaar",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
				"usernameTaken": "gebruikersnaam niet beschikbaar",
```

File: kofta/public/locales/nl/translation.json [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+		"addToCalendar": {
+			"add": "Toevoegen aan kalender"
+		},
+		"wsKilled": {
+			"description": "Webcontact is gestopt door de server. Dit gebeurd meestal als je een andere website bezoekt in een ander venster.",
+			"reconnect": "opnieuw verbinding maken"
+		},
+		"modals": {
+			"createRoomModal": {
+				"public": "publiek",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
				"public": "openbaar",
```
publiekelijk would also be a good translation

File: kofta/public/locales/nl/translation.json [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+		"micPermissionBanner": {
+			"permissionDenied": "Toestemming geweigerd om toegang te krijgen tot je microfoon (mogelijk moet je naar de browserinstellingen gaan en de pagina opnieuw laden)",
+			"dismiss": "Sluiten",
+			"tryAgain": "probeer het opnieuw"
+		},
+		"keyboardShortcuts": {
+			"setKeybind": "zet toetsbinding",
+			"listening": "aan het luisteren",
+			"toggleMuteKeybind": "Microfoon dempen schakelen toetsbinding",
+			"togglePushToTalkKeybind": "druk-om-te-praten toetsbinding"
```
Comment (thomascrmbz) [inline kofta/public/locales/nl/translation.json]:
```suggestion
			"setKeybind": "zet keybind",
			"listening": "aan het luisteren",
			"toggleMuteKeybind": "Microfoon dempen schakelen keybind",
			"togglePushToTalkKeybind": "druk-om-te-praten keybind"
```

change "toetsbinding" to "keybind" or "toetscombinatie"
I'm dutch and I've never heard of a "toetsbinding"

File: kofta/public/locales/nl/translation.json [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+			"mute": "Demp",
+			"unmute": "Dempen opheffen",
+			"makeRoomPublicBtn": "Kamer publiek maken!",
+			"settings": "Instellingen",
+			"speaker": "Luidspreker",
+			"listener": "Luisteraar"
+		},
+		"deviceNotSupported": {
+			"notSupported": "Je toestel is momenteel niet ondersteund. Maak een",
+			"linkText": "kwestie aan op GitHub",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
			"linkText": "issue aan op GitHub",
```
It's always called an issue on Github

File: kofta/public/locales/nl/translation.json [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+			"confirmLeaveRoom": "Weet je het zeker dat je deze kamer wilt verlaten?",
+			"leave": "Verlaten",
+			"inviteUsersToRoomBtn": "Gebruikers uitnodigen",
+			"invite": "Uitnodigen",
+			"toggleMuteMicBtn": "Microfoon dempen schakelen",
+			"mute": "Demp",
+			"unmute": "Dempen opheffen",
+			"makeRoomPublicBtn": "Kamer publiek maken!",
+			"settings": "Instellingen",
+			"speaker": "Luidspreker",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json:115]:
```suggestion
			"speaker": "Spreker",
```

File: kofta/public/locales/nl/translation.json [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+		"bottomVoiceControl": {
+			"leaveCurrentRoomBtn": "Huidige kamer verlaten",
+			"confirmLeaveRoom": "Weet je het zeker dat je deze kamer wilt verlaten?",
+			"leave": "Verlaten",
+			"inviteUsersToRoomBtn": "Gebruikers uitnodigen",
+			"invite": "Uitnodigen",
+			"toggleMuteMicBtn": "Microfoon dempen schakelen",
+			"mute": "Demp",
+			"unmute": "Dempen opheffen",
+			"makeRoomPublicBtn": "Kamer publiek maken!",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
			"makeRoomPublicBtn": "Kamer openbaar maken!",
```

**Review by mrperson123 — CHANGES_REQUESTED** [review]
Some translations weren't really correct.

File: kofta/public/locales/nl/translation.json [hunk 14]
```diff
@@ -0,0 +1,222 @@
...
+			"togglePushToTalkKeybind": "druk-om-te-praten toetsbinding"
+		},
+		"userVolumeSlider": {
+			"noAudioMessage": "geen mogelijkheid om geluid te ontvangen door een onbekende reden"
+		},
+		"addToCalendar": {
+			"add": "Toevoegen aan kalender"
+		},
+		"wsKilled": {
+			"description": "Webcontact is gestopt door de server. Dit gebeurd meestal als je een andere website bezoekt in een ander venster.",
```
Comment (thomascrmbz) [inline kofta/public/locales/nl/translation.json]:
```suggestion
			"description": "WebSocket is gestopt door de server. Dit gebeurt meestal wanneer je de website in een ander tabblad opent.",
```

don't translate WebSocket, it's a technology, there is no dutch translation for that
"Webcontact" is the literal translation of ("Web" + "Socket") -> ("Web" + "contact")

also the translation is not what the english sentence means

**Comment by ThijmenGThN** [comment]
Thanks for the corrections @mrperson123

File: kofta/public/locales/nl/translation.json [hunk 15]
```diff
@@ -0,0 +1,222 @@
...
+				"likeToJoin": ", wil je deelnemen?",
+				"inviteReceived": "je bent uitgenodigd voor"
+			},
+			"editProfileModal": {
+				"usernameTaken": "gebruikers naam niet beschikbaar",
+				"avatarUrlError": "Ongeldige foto",
+				"avatarUrlLabel": "Github/Twitter avatar link",
+				"displayNameError": "lengte 2 tot 50 karakters",
+				"displayNameLabel": "Weergavenaam",
+				"usernameError": "lengte 4 tot 15 tekens en alleen alfanumeriek / onderstrepingsteken",
```
Comment (thomascrmbz) [inline kofta/public/locales/nl/translation.json]:
```suggestion
				"usernameError": "lengte 4 tot 15 tekens en alleen alfanumeriek / underscore",
```

change "onderstrepingsteken" to "underscore"
again i've never heard of an "onderstrepingsteken"

File: kofta/public/locales/nl/translation.json [hunk 16]
```diff
@@ -0,0 +1,222 @@
...
+			"description": "Webcontact is gestopt door de server. Dit gebeurd meestal als je een andere website bezoekt in een ander venster.",
+			"reconnect": "opnieuw verbinding maken"
+		},
+		"modals": {
+			"createRoomModal": {
+				"public": "publiek",
+				"private": "privé",
+				"roomName": "kamer naam",
+				"roomDescription": "kamer beschrijving",
+				"descriptionError": "maximale grote 500",
```
Comment (thomascrmbz) [inline kofta/public/locales/nl/translation.json]:
```suggestion
				"descriptionError": "maximale lengte 500",
```

change "grote" to "lengte" (on line 172 it's translated as "lengte")

File: kofta/public/locales/nl/translation.json [hunk 17]
```diff
@@ -0,0 +1,222 @@
...
+				"avatarUrlLabel": "Github/Twitter avatar link",
+				"displayNameError": "lengte 2 tot 50 karakters",
+				"displayNameLabel": "Weergavenaam",
+				"usernameError": "lengte 4 tot 15 tekens en alleen alfanumeriek / onderstrepingsteken",
+				"usernameLabel": "Gebruikersnaam",
+				"bioError": "maximale lengte van 160 tekens",
+				"bioLabel": "Bio"
+			},
+			"profileModal": {
+				"blockUserConfirm": "Weet je zeker dat je wilt voorkomen dat deze gebruiker lid wordt van een kamer die je ooit hebt gemaakt?",
```
Comment (thomascrmbz) [inline kofta/public/locales/nl/translation.json]:
```suggestion
				"blockUserConfirm": "Weet je zeker dat je deze gebruiker wil blokkeren om deel te nemen aan elke kamer die je ooit zal aanmaken?",
```

this is a translation in the past tense ("ooit hebt gemaakt")
the original sentence is "you ever create" so the correct translation is "ooit zal maken"

File: kofta/public/locales/nl/translation.json [hunk 18]
```diff
@@ -0,0 +1,222 @@
...
+		"roomChat": {
+			"title": "Chat",
+			"emotesSoon": "[emoticons binnenkort]",
+			"bannedAlert": "Je bent verbannen uit de chat",
+			"waitAlert": "U moet een seconde wachten voordat u weer een bericht verzendt",
+			"search": "Zoeken",
+			"searchResults": "Zoeken Resultaten",
+			"recent": "Vaak Gebruikt",
+			"sendMessage": "Een Bericht Versturen",
+			"whisper": "Vluisteren",
```
Comment (mrperson123) [inline kofta/public/locales/nl/translation.json]:
```suggestion
			"whisper": "Fluisteren",
```

**Review by mrperson123 — CHANGES_REQUESTED** [review]

File: kofta/public/locales/nl/translation.json [hunk 19]
```diff
@@ -0,0 +1,222 @@
...
+			"soundSettings": "naar geluids-instellingen",
+			"deleteAccount": "account verwijderen"
+		},
+		"notFound": {
+			"whoopsError": "Oops! Deze pagina is kwijtgeraakt in een conversatie.",
+			"goHomeMessage": "Geen zorgen. Jij kan het",
+			"goHomeLinkText": "naar thuispagina"
+		},
+		"room": {
+			"speakers": "Luidsprekers",
```
Comment (thomascrmbz) [inline kofta/public/locales/nl/translation.json]:
```suggestion
			"speakers": "Sprekers",
```

**Comment by mrperson123** [comment]
@ThijmenGThN why don't you commit the suggestions?

**Comment by ThijmenGThN** [comment]
@mrperson123 I will, just a moment please.

File: kofta/public/locales/nl/translation.json (hunk 12 above)
Comment (ThijmenGThN) [reply kofta/public/locales/nl/translation.json:115]: This translation does not sound quite right.

File: kofta/public/locales/nl/translation.json (hunk 5 above)
Comment (ThijmenGThN) [reply kofta/public/locales/nl/translation.json:180]: This translation does not sound quite right.

**Pull request closed without merging**